### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Olivier-Kango**
Hi there 👋,

In this particular project, I : 

- Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first. (Note: I ignore login logic, I am using Devise in coming milestone)
    
blog_app_1
blog_app_2

- Use methods that I have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- **Hint:** In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,34 @@
-<h1>Posts #index</h1>
-<h2>This is the posts index page</h2>
\ No newline at end of file
+<div class="main-container">
+  <div class="users-wrapper flex">
+    <div class="img-cont">
+      <%= image_tag @user.photo, :size => "50x50", alt: "user photo" %>
```
Comment (cynthiainga) [inline app/views/posts/index.html.erb]: - [ ] Good job so far, but you need to work a little bit on your design. Please make sure the images have borders like in the wireframe 🙃 😉 

Your design
Screen Shot 2023-02-02 at 14 45 38

Wireframe 
Screen Shot 2023-02-02 at 14 46 00

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,72 @@
...
+  height: 100vh;
+}
+
+.main-container {
+  width: 60%;
+  margin-top: 2%;
+  gap: 5%;
+  flex-direction: column;
+  height: 100vh;
+}
```
Comment (cynthiainga) [inline app/assets/stylesheets/application.css]: - [ ]  Good job so far, but you need to work a little bit on your design. You can see the parts that you need to work on in the screenshot below 👇 

Your design

Screen Shot 2023-02-02 at 14 50 20

Wireframe

Screen Shot 2023-02-02 at 14 50 32

- [ ] Also, don't forget to work a little bit on your design for the see-all post page 😉 😉 

Your design

Screen Shot 2023-02-02 at 15 04 27

Wireframe

Screen Shot 2023-02-02 at 15 04 44

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Olivier-Kango, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- I think it would be better if you update your MIT license file 🙃 🙃 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Olivier-Kango** [comment]
Kidnly try to use this link in order to give a good image. photo =
https://user-images.githubusercontent.com/108806646/216281530-fa5c3020-197a-498e-9024-339268ba1959.jpg

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Olivier-Kango 👋, 

Well done for implementing the required changes 👏 
Your project is now complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**